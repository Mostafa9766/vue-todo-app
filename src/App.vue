<template>
  <div>
    <Dashboard></Dashboard>
  </div>
</template>

<script>
import { mapActions } from 'vuex';
import Dashboard from './components/DashboardComponent.vue';

export default {
  components: { Dashboard },
  mounted() {
    this.initData();
  },
  methods: {
    ...mapActions(['initData']),
  },
};

</script>
