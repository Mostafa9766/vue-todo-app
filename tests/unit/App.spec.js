import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import ElementUi from 'element-ui';
import App from '@/App.vue';

describe('App.vue', () => {
  const localVue = createLocalVue();
  localVue.use(Vuex);
  localVue.use(ElementUi);

  let store;

  beforeEach(() => {
    store = new Vuex.Store({});
  });

  it('render App component', () => {
    const wrapper = shallowMount(App, {
      localVue,
      store,
    });

    expect(wrapper).toBeTruthy();
  });
});
