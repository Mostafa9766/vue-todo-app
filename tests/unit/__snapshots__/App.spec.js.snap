// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue render App component 1`] = `
<div>
  <dashboard-stub></dashboard-stub>
</div>
`;
